src/client/src/database.rs

```diff
@@ -111,6 +131,14 @@ impl Database {
         let request = Ticket {
             ticket: request.encode_to_vec(),
         };
+        let mut request = tonic::Request::new(request);
```

Comment:
Maybe it's better to put auth in RequestHeader? 

https://github.com/GreptimeTeam/greptime-proto/blob/196bb742fba063b7e3a7a960c55bf73062c9501e/proto/greptime/v1/database.proto#L12

I think Tonic metadata should contain only grpc related properties, like flow control or metrics.

Reply:
I'm more than happy to put auth info here if its ok for you 🤣 saves the repeated `FlightContext` param too as @sunng87 mentioned above

Reply:
can we make `string authorization = 3;` a seperate message, so we can use `None` in request header and make authorization more flexible

Reply:
> can we make string authorization = 3; a seperate message

Sounds good to me.